django-testbed/django-root/xenopus_frog/views.py

```diff
@@ -7,14 +7,57 @@
 from xenopus_frog.models import User
 
 
+class BunniesSerializer(ModelSerializer):
```

Comment:
🐰

Reply:
Is there a reason we wouldn't put this in the library?

Reply:
I think maybe we shouldn't extend anything? just use as a mixin that assumes `self.fields` and `self.Meta` exists